### PR TITLE
Set the filesystem root on zypper when given

### PR DESCRIPTION
**Pull request by mssola**
Fixes bsc#1230229

### CODE REVIEW

**Comment by mssola** [comment]
[[L3 P2] SLE Micro 5.5 unable to register with new suseconnect-ng 1.11](https://trello.com/c/ODCPKfYE/3543-l3-p2-sle-micro-55-unable-to-register-with-new-suseconnect-ng-111)

**Comment by admd** [comment]
Hi @mssola thank you for the fix. When can we expect this fix to be released?

**Comment by mssola** [comment]
> Hi @mssola thank you for the fix. When can we expect this fix to be released?

Well, as soon as it gets merged we can contact maintenance to create a PTF for it. Otherwise we could submit a patch-level release, but this is a bit unusual for this project.

**Comment by admd** [comment]
thank you, Miquel. This will affect every customer out there so PTF route will not work. We will need to release the fix for everyone.

**Comment by deneb-alpha** [comment]
hi @mssola,
in addition to what @admd already mention, if you want to re-prioritize the maintenance request for `suseconnect-ng` you can report a  MSC ticket for maintenance via https://jira.suse.com/secure/CreateIssue.jspa?&pid=24490

I also dropped a message to them here too: https://suse.slack.com/archives/C02CCRM8946/p1726220452158809

File: build/packaging/suseconnect-ng.changes [hunk 1]
```diff
@@ -1,7 +1,8 @@
 -------------------------------------------------------------------
-Thu Jul 11 12:36:22 UTC 2024 - Miquel Sabate Sola <msabate@suse.com>
+Fri Sep 13 14:11:22 UTC 2024 - Miquel Sabate Sola <msabate@suse.com>
 
-- UNDER DEVELOPMENT: version 1.12
+- Update version to 1.12:
+  - Set the filesystem root on zypper when given (bsc#1230229)
```
Comment (ngetahun) [inline build/packaging/suseconnect-ng.changes:5]:
```suggestion
  - Set the filesystem root on zypper when given (bsc#1230229,bsc#1229014)
```

**Review by ngetahun — APPROVED** [review]
LGTM!

**Comment by mssola** [comment]
[SUSEConnect 1.12 release](https://trello.com/c/LsYY0Vy9/3548-suseconnect-112-release)

**Comment by mssola** [comment]
@admd @deneb-alpha everything has been pushed for a 1.12 release on maintenance.

**Comment by deneb-alpha** [comment]
> @admd @deneb-alpha everything has been pushed for a 1.12 release on maintenance.

I left a comment: 
https://bugzilla.suse.com/show_bug.cgi?id=1230229#c9

**Pull request merged**